docs/user/function-support.rst

```diff
@@ -0,0 +1,117 @@
+Function support
+=====================
+
+Aggregate Functions
+--------------------------------------
+**avg(x)**
```

Comment:
For information like this, which is almost common info for SQL, we may can put them as reference? like https://openmldb.ai/docs/zh/main/reference/index.html.